SignalServiceKit/src/Contacts/SignalAccount.m

```diff
@@ -50,16 +50,9 @@ - (nullable NSString *)uniqueId
     return _recipientId;
 }
 
-- (NSString *)displayName
+- (nullable NSString *)contactFullName
 {
-    NSString *baseName = (self.contact.fullName.length > 0 ? self.contact.fullName : self.recipientId);
-
-    OWSAssert(self.hasMultipleAccountContact == (self.multipleAccountLabelText != nil));
-    NSString *displayName = (self.multipleAccountLabelText
-            ? [NSString stringWithFormat:@"%@ (%@)", baseName, self.multipleAccountLabelText]
-            : baseName);
-
-    return displayName.filterStringForDisplay;
+    return self.contact.fullName.filterStringForDisplay;
```

Comment:
FYI I like to check `length non-zero` of these filtered strings rather than nil.

Reply:
ok - updated. PTAL @charlesmchen